### PR TITLE
My work

### PR DESCRIPTION
**Pull request by Mutalenic**
- add HTML, CSS, and readme files
- configure linters
- use GitHub flow

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,63 +1,147 @@
...
+         <li><a href="#"><i class="fa fa-github bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-twitter bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-linkedin bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-angellist bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-medium bars"></i></a></li>
+     </ul>
+</section>      
+
+  <section class="header">
+    <h2 class="recent">MY RECENT WORK</h2>
```
Comment (ahmetbozaci) [inline index.html:69]: - [ ] Before this text, you miss worktext with an arrow. Please add it.
### ORIGINAL
image

File: style.css [hunk 2]
```diff
@@ -121,3 +120,96 @@
...
+
+.post {
+  color: #fff;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 32px;
+}
+
+.lang {
+  display: flex;
```
Comment (ahmetbozaci) [inline style.css:183]: ### OPTIONAL
- [ ] For DRY CSS you should combine duplicate properties. Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.Flex is just an example you can do it to duplicate properties.



- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: index.html [hunk 3]
```diff
@@ -1,63 +1,147 @@
...
+              <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups
+                required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+              <ul class="lang">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+              </ul>
+              <button class="button" type="button">See Project</button>
```
Comment (ahmetbozaci) [inline index.html:139]: - [ ] The button should have the button styling: Active, Hover, and Pressed - Please add it all buttons. [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Implement%20the%20button%20interactions%20(enable%2C%20hover%2C%20etc.).)
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️

### Hello @Mutalenic  ✋🏼 
#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- You stick to the design.✔️
- You know how to use flexbox very well.✔️
- You use HTML semantic elements correctly.✔️

___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] Your commit messages should always start with capital letters and you should write an imperative tense. I see some of them are not like that. I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this for your next projects.
Instead of writing `'Added style' or Created the file`' you should write `Add style'  or 'Create the file'`

**OPTIONAL**
- [ ] You can change them by using `git rebase`. If you want to know how to change ı leave a few resources. 
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)

___

### Cheers and Happy coding! 💻 👏

almost

___

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: index.html (hunk 1 above)
Comment (mateo951) [reply index.html:69]: Addressed 🔥

File: index.html (hunk 3 above)
Comment (mateo951) [reply index.html:139]: Addressed 🔥

File: index.html [hunk 4]
```diff
@@ -1,63 +1,148 @@
...
+     <hr class="line">
+     <ul class="icons">
+         <li><a href="#"><i class="fa fa-github bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-twitter bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-linkedin bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-angellist bars"></i></a></li>
+         <li><a href="#"><i class="fa fa-medium bars"></i></a></li>
+     </ul>
+     <div><span>&#8595; Work</span></div>
+</section>
```
Comment (mateo951) [inline index.html:66]: #### OPTIONAL

This is not part of the scope of this review so I'll mark it as a suggestion so you can implement it if you wish 🧐

There are some issues with the margins in this section as you will see in the following image 😥 Some gaps are appearing on the sides of the browser screen so you can find a way to fix it so your portfolio looks even more 🔥🔥🔥

image

**Review by mateo951 — APPROVED** [review]
Hi @Mutalenic, 

Great job implementing all the changes required by the previous reviewer ✔🙌🚀 Your project is complete!  There is nothing else to say other than... it's time to merge it 👌

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**